src/helpers/ignoreFilesPattern.ts

```diff
@@ -4,13 +4,13 @@ import fs from 'fs-extra';
 import { formatHomePath } from '../helpers/formatHomePath';
 import type { Site } from '../types';
 
-export const excludePatterns = ['conf'];
+export const excludePatterns = ['conf', 'logs'];
 
 export const localBackupsIgnoreFileName = '.localbackupaddonignore.txt';
 
 // Returns an array of directories within the site folder that we want to include in the backup
-// Filters out the 'conf' directory and any `.` files
```

Comment:
This documentation was inaccurate - we're explicitly including dotfiles, and honestly might not even need the second glob pattern?